indico/modules/attachments/controllers/management/event.py

```diff
@@ -97,3 +104,20 @@ class RHPackageEventAttachmentsManagement(AttachmentPackageMixin, RHManageEventB
     wp = WPPackageEventAttachmentsManagement
     management = True
     ALLOW_LOCKED = True
+
+
+class RHSetUploadPermissions(RHManageEventBase):
+    def _process(self):
+        current_settings = attachments_settings.get_all(self.event)
+        form = SetUploadPermissionsForm(obj=FormDefaults(**current_settings))
+        if form.validate_on_submit():
+            attachments_settings.set_multi(self.event, form.data)
+            new_settings = attachments_settings.get_all(self.event)
+            flash(_('Upload permissions settings changed successfully'), 'success')
+            changes = {k: (v, new_settings[k]) for k, v in current_settings.items() if v != new_settings[k]}
+            if changes:
+                log_fields = {'managers_only': 'Only managers can upload attachments'}
+                self.event.log(EventLogRealm.management, LogKind.change, 'Attachments', 'Settings updated',
```

Comment:
"Upload settings updated"? That way you can use "Managers only" in the log fields above to keep that one more concise.